### DISCUSSION TITLE
how to install the nightly version in fedora37 or when will some features be included in the stable version?

### DISCUSSION THREAD

**Discussion by vladQll** [xournalpp/xournalpp#4711] [General]
Hi, I'm on fedora, and in the store is only the stable version, I would like to know when would be moving some features from the night version to the stable version, such as the double arrow figure, fill options, etc, as I did not find a proper guide on how to install the night version in fedora 37 and I did not see any package .rpm thanks

**Comment by rolandlo** [comment]
You can try the AppImage from the [nightly build](https://github.com/xournalpp/xournalpp/releases/tag/nightly). 
It shouldn't take longer than end of March until the current nightly version becomes the new stable version.

**Reply by vladQll** [reply]
hello when the nightly version becomes the stable version, it will only be necessary for me to do an update something I usually do is `sudo dnf -y update` or I will have to uninstall and reinstall, for the moment I will use the stable version.

**Comment by rolandlo** [comment]
`sudo dnf -y update` should be enough.
